lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -561,6 +562,7 @@ def build_module(self, trans, payload={}):
         Builds module models for the workflow editor.
         """
         inputs = payload.get('inputs', {})
+        trans.workflow_build_mode = workflow_building_modes.ENABLED
```

Comment:
@mvdbeek Did you mean `trans.workflow_building_mode` ?

Reply:
Exactly, and was half expecting this wouldn't work because we may also use this when building the workflow run form. Which it may not after all. Thanks for checking @nsoranzo !